libdevice/sanitizer_utils.cpp

```diff
@@ -589,6 +623,8 @@ inline uptr __asan_region_is_poisoned(uptr beg, uint32_t as, size_t size) {
   return 0;
 }
 
+constexpr size_t AlignMask(size_t n) { return n - 1; }
```

Comment:
add a assertion here to make sure n is a power of 2?

Reply:
Done